stores-api/transaction-api/src/main/java/com/bloxbean/cardano/yaci/store/api/transaction/controller/TransactionController.java

```diff
@@ -66,4 +74,59 @@ public TransactionPage getTransactions(@RequestParam(name = "page", defaultValue
     public List<TxnWitness> getTransactionWitnesses(@PathVariable @Pattern(regexp = "^[0-9a-fA-F]{64}$") String txHash) {
         return transactionService.getTransactionWitnesses(txHash);
     }
+    
+    @GetMapping(value = "{txHash}/cbor", produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
```

Comment:
Should we also enable these endpoints to return cbor hex string? A separate endpoint is ok. 
Alternative, if possible, if client sends Accept header as `application\json`, we can wrap the cbor hex string in a json and return 

`{ "cborHex": "84a200581c1c7b4a9e..." }`

Reply:
Same for BlockController